modules/core/public/assets/js/loginuserpass.js

```diff
@@ -1,12 +1,16 @@
 ready(function () {
     var button = document.getElementById("submit_button");
```

Comment:
Might be a bit nitpicky, but this is now unused and can also be removed

Reply:
Actually, no, I removed the wrong one! We need one for `form`